utils/media.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'styled-components';
+import { RuleSet } from 'styled-components/dist/types';
+
+export const mobile = (inner: ReturnType<typeof css>): RuleSet<object> => css`
```

Comment:
You should be able to remove `RuleSet<object>` for these and use the same `ReturnType<typeof css>` annotation. Perhaps store it to a local type so you can reuse it?

```ts
type CSSUtilType = ReturnType<typeof css>
```